src/board.c

```diff
@@ -504,7 +411,7 @@ void Board_LTC6804_GetCellTemperatures(BMS_PACK_STATUS_T * pack_status, uint8_t
 #ifndef TEST_HARDWARE
 #ifdef FSAE_DRIVERS
     if ((msTicks - board_lastThermistorShiftTime_ms) > TIME_PER_THERMISTOR_MS) {
-        board_lastThermistorShiftTime_ms += TIME_PER_THERMISTOR_MS;
+        board_lastThermistorShiftTime_ms = msTicks;
```

Comment:
Also I think this one wins the award for hardest-bug-to-find-in-real-testing cause it would have just caused the thermistors to slooowly read faster and faster until it did two back to back reads and looped back around lol